File: client/src/app/model/Pass.ts
import {Client} from './Client';
import {Flight} from './Flight';
import {Country} from './Country';
import {Resort} from './Resort';
import {Hotel} from './Hotel';

export class Pass{
  public id:number;
  public transport: string;
  public change:boolean = false;
  public client:Client;
  public flight:Flight;
  public country:Country;
  public resort:Resort;
  public hotel:Hotel;


}

File: client/src/main/java/tour/tour/entity/Pass.java
package tour.tour.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.RequiredArgsConstructor;

import javax.persistence.*;

@Entity
@Data
@RequiredArgsConstructor
@AllArgsConstructor
public class Pass {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String transport;
    private transient boolean change;
    @ManyToOne
    private Client client;
    @ManyToOne
    private Flight flight;
    @ManyToOne
    private Country country;
    @ManyToOne
    private Resort resort;
    @ManyToOne
    private Hotel hotel;

}
